kin-devplatform-sdk/src/main/java/kin/devplatform/KinEcosystemInitiator.java

```diff
@@ -63,10 +67,10 @@ private KinEcosystemInitiator() {
 	 */
 	public void internalInit(Context context) {
 		try {
-			init(context, null);
+			init(context, null, null, false, null, null);
```

Comment:
do you have more nulls to put in here?

Reply:
I have many more nulls in the shelf but i wanted to keep some for me 🥇 .
Because this method is only used in case of process restart and the design didn't considered that it could be async, which is how it is implemented now because of the migration, so we will anyway change it a bit and will try to overcome this ugliness.